source/postprocess/visualization/maximum_horizontal_compressive_stress.cc

```diff
@@ -111,7 +111,7 @@ namespace aspect
                   const double b = orthogonal_directions[1] *
                                    (compressive_stress *
                                     orthogonal_directions[1]);
-                  const double c = orthogonal_directions[0] *
+                  const double c = 2.0*orthogonal_directions[0] *
```

Comment:
The comment above this block references the formulas in the documentation. Do they also have the factor of 2?

Does this change affect the output of any tests?

Reply:
This is #2833, right?